src/signals/incident-management/containers/IncidentDetail/components/Detail/components/ExtraProperties/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { Fragment } from 'react';
+import type { FunctionComponent } from 'react';
+import type { Answer, MappedLegacyItem, Item, CheckboxInput, ExtraPropertiesTypes, LegacyAnswer } from './types';
+
+const getValue = (answer: Answer | LegacyAnswer): string | JSX.Element[] => {
+  if (Array.isArray(answer)) {
+    return answer.map((item, index) => {
+      if (typeof item !== 'string') {
+        if (item.description) {
+          return <div key={item.id}>{`${item.description}${item.id && ` - ${item.id}`}`}</div>;
+        }
+      }
+
+      // eslint-disable-next-line react/no-array-index-key
+      return <div key={index}>{item}</div>;
```

Comment:
It is not recommended to use index as key in react lists. If you would use the item as a key here, the eslint-disable won't be needed.